src/components/pullRequests/index.tsx

```diff
@@ -19,68 +19,34 @@ const pullRequest: FC<prData> = ({
   const created = new Date(createdAt);
   const updated = new Date(updatedAt);
   const presentDate = new Date();
+
   const createdAgo = `${Math.ceil(
     (presentDate.getTime() - created.getTime()) / (1000 * 3600 * 24),
   )}days ago`;
   const updatedAgo = `${Math.ceil(
     (presentDate.getTime() - updated.getTime()) / (1000 * 3600 * 24),
   )}days ago`;
-  const [createdTime, updateCreatedTime] = useState(createdAgo);
-  const [updatedTime, updateUpdatedTime] = useState(updatedAgo);
+
+  const prData = {
+    CreatedBy: username,
+    Created: createdAgo,
+    Updated: updatedAgo,
+  };
+  const PR:any[] = [];
+  function getPr() {
+    // eslint-disable-next-line array-callback-return
```

Comment:
arrow function? Also please get rid of commented code.

Reply:
Converted it to arrow function and used forEach instead of map to get rid of this comment.

https://github.com/Real-Dev-Squad/website-status/commit/f84f9afa7df2679012784eb4f1b1558a2496fdb4